Makefile

```diff
@@ -97,7 +98,7 @@ $(STATIK):
 	$(GO) get -u github.com/rakyll/statik
 
 $(GORELEASER):
```

Comment:
We can drop this block and rather move `bin/goreleaser/v0.142.0/$(OS)/goreleaser` up to the `GORELEASER := bin/goreleaser/v0.142.0/$(OS)/goreleaser` statement.

Reply:
Done :+1: